### DISCUSSION TITLE
Trying to execute json file

### DISCUSSION THREAD

**Discussion by dharmarajrdr** [eslint/eslint#15947] [General] [answered]
### Environment

Node version: 
npm version: 
Local ESLint version: 
Global ESLint version: 
Operating System: 

### What parser are you using?

Default (Espree)

### What did you do?

module.exports = {
    meta: { ... },
    create: function(context) {
        // declare the state of the rule
console.log(context);
        return {
            ReturnStatement: function(node) {
                // at a ReturnStatement node while going down
            },
            // at a function expression node while going up:
            "FunctionExpression:exit": checkLastSegment,
            "ArrowFunctionExpression:exit": checkLastSegment,
            onCodePathStart: function (codePath, node) {
                // at the start of analyzing a code path
            },
            onCodePathEnd: function(codePath, node) {
                // at the end of analyzing a code path
            }
        };
    }
};

### What did you expect to happen?

While executing json file, it shows 
 2:27  error  Parsing error: Unexpected token :

### What actually happened?

Why the json content present inside the sample.json file is not creating context, when I run as : eslint sample.json --rulesdir /rules 

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by aladdin-add** [comment, marked as answer]
it's expected. eslint is to lint js, while a json is not a valid js.
